### PR TITLE
feat: Document "Fix issues" button

### PR DESCRIPTION
**Pull request by nicklem**
Document the "Fix issues" button on the Issues and PR pages.

NOTE: this PR has no associated Jira task. See [Slack thread](https://codacy.slack.com/archives/C015DHTSJUR/p1716987582202659) for more context.

### :eyes: Live preview
* https://feat-patchemons--docs-codacy.netlify.app/repositories/issues/#fixing-issues-automatically
* https://feat-patchemons--docs-codacy.netlify.app/repositories/pull-requests/#fixing-issues-automatically
* https://feat-patchemons--docs-codacy.netlify.app/repositories/repository-dashboard/#issues-breakdown

### CODE REVIEW

**Review by joanasteodoro — COMMENTED** [review]
we also have this button on the repository dashboard!
thank you, Nico, for putting your efforts on updating the docs so quickly ❤️

**Review by andrzej-janczak — APPROVED** [review]
Great! Thank you 🙇

File: docs/repositories/pull-requests.md [hunk 1]
```diff
@@ -47,6 +47,12 @@ Click a specific commit to see [detailed information about that commit](commits.
 
 ![Commits tab](images/pull-requests-tab-commits.png)
 
+## Fixing issues automatically
+
+!!! info "This section applies to GitHub repositories only"
+
+If Codacy detects code patterns that have suggested fixes, you can apply the fixes directly by clicking the **Fix issues** button that appears above the issue list. This action applies the suggested fixes and commits the changes to the pull request.
```
Comment (claudiacarpinteiro) [inline docs/repositories/pull-requests.md:54]: Same here. Does the following make sense?

```suggestion
If Codacy detects code patterns that have suggested fixes, you can apply those fixes directly by clicking the **Fix issues** button above the issue list. This action commits the suggested fixes to the pull request.
```

**Review by claudiacarpinteiro — COMMENTED** [review]
@nicklem I left a couple of comments for your evaluation. 🙏

**Pull request merged**